src/RenderItems.js

```diff
@@ -1,13 +1,51 @@
 import React, { Component } from 'react'
-import Data from './Data.js'
+// import Data from 'Data.js'
 import Item from './Item.js'
 
+
+
 export default class RenderItems extends Component {
+    state = {
+        filter: ''
+    }
+
+    keySet = [...new Set(this.props.Data.map(item => item.keyword))]
```

Comment:
This works! (and is how I solved this) One drawback, it's not very readable. You could add this this.props.Data.map(item => item.keyword) to be a variable that's something like keywordArray. That way when you Set() it would read more like English. It's a little extra code but makes it way more maintainable in the future when you haven't seen this code in few months and go back to it. Great work!